app/controllers/activejob_web/jobs_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module ActivejobWeb
+  class JobsController < ApplicationController
+    before_action :set_job, only: %i[show edit update]
+
+    def index
+      @jobs = ActivejobWeb::Job.includes(:executors).where(activejob_web_job_executors: { executor_id: activejob_web_current_user.id })
+    end
+
+    def show
+      if @job.executors.include?(activejob_web_current_user)
+        render :show
+      else
+        redirect_to root_path
+        flash[:notice] = 'You are not authorized to perform this action.'
+      end
+    end
+
+    def edit; end
+
+    def update
+      @job.approver_ids = job_params[:approver_ids]
```

Comment:
@GowthamK01 Instead of assigning these params one by one. Can we mass assign using the below?


```
@job.update(job_params)
```

Reply:
@gomathi-mallow Have checked and updated with `job_params`